src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/SoftwareChoicesStepDef.scala

```diff
@@ -35,11 +35,12 @@ class SoftwareChoicesStepDef extends BaseStepDef {
 
   When("""^I enter '(.*)' into the search bar$""") { (searchTerm: String) =>
     write(searchBarId, searchTerm)
-  }
-
-  And("""^I wait for the software vendor list to update$""") { () =>
-    fluentWait
-      .until(ExpectedConditions.presenceOfElementLocated(By.id(softwareVendorsId)))
+    fluentWait.until(
+      ExpectedConditions.presenceOfElementLocated(By.cssSelector(".loading"))
+    )
+    fluentWait.until(
+      ExpectedConditions.invisibilityOfElementLocated(By.cssSelector(".loading"))
+    )
```

Comment:
This step fails on my computer. We could wait for the software providers to be visible again.
```suggestion
    fluentWait
      .until(
        ExpectedConditions.visibilityOfElementLocated(By.id(softwareVendorsId))
      )
```